test/article.test.js

```diff
@@ -2,16 +2,51 @@ const chai = require('chai')
 const chaiHttp = require('chai-http')
 const app = require('../server.js')
 
-chai.should()
-chai.use(chaiHttp)
+const should = chai.should()
 
-function checkResponse(res) {
+function postVote(id, vote, old) {
+  return new Promise((resolve) => {
+    chai.request(app)
+            .post('/articles/vote')
+            .send({ id, vote })
+            .end((err, res) => {
+              resolve({ res, old })
```

Comment:
This is quite a hacky way to pass down `old` in the promise, make a variable for it instead